### PR TITLE
[RI2] Dos2unix chapter 1, 3, 4 and 5

### PR DESCRIPTION
**Pull request by collivier**
It basically fails the RST conversion.

Signed-off-by: Cédric Ollivier <cedric.ollivier@orange.com>

### CODE REVIEW

File: doc/ref_impl/cntt-ri2/chapters/chapter01.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿# 1. Overview
+# 1. Overview
 
 ![State](../figures/bogo_dfp.png) <!-- width="35" -->
```
Comment (pgoyal01) [inline doc/ref_impl/cntt-ri2/chapters/chapter01.md:3]: I thought we agreed to delete the bogometers

Comment (gkunz) [reply doc/ref_impl/cntt-ri2/chapters/chapter01.md:3]: @pgoyal01 we did. They have been removed in PR #2656 which implements the rst conversion for RI2.

**Comment by collivier** [comment]
Fully agree to remove it in a second change.
Here the proposal is to remove the dos hidden chars which fails the conversion.

**Review by pgoyal01 — APPROVED** [review]

**Comment by gkunz** [comment]
@collivier there is a PR #2656 which implements the rst conversion. I'll check this PR for hidden characters, but those files have been processed by pandoc, so I don't expect any.

**Comment by collivier** [comment]
@gkunz The pandoc conversions fails here (and atom doesn't interpret this lines as title) - double checking
I would encourage to fix the md first (agree that pandoc should remove them automatically).

**Comment by gkunz** [comment]
@collivier I checked the files in the PR mentioned above, but pandoc has, as we assumed, unified the file encoding. At least running dos2unix does not change the files. I also checked the resulting output, but couldn't spot issues with the headings. Maybe you can quickly take a look and check if the headings are correct in my PR since you have encountered problems during conversion. Thanks.

**Comment by collivier** [comment]
@gkunz Could we merge it? trivial but crucial.

**Comment by gkunz** [comment]
@collivier we can merge it, but it won't matter. See #2656. I'd rather like to merge #2656.

**Comment by collivier** [comment]
@gkunz yes it will be removed by your next change - but the point is to apply the change as we don't know when rst will be effective.

**Comment by gkunz** [comment]
The problem is that I cannot merge it. @rihabbanday?

**Review by rihabbanday — APPROVED** [review]

**Pull request merged**